base/compiler/ssair/passes.jl

```diff
@@ -1132,19 +1137,35 @@ function adce_pass!(ir::IRCode)
     end
     non_dce_finish!(compact)
     for phi in all_phis
-        count_uses(compact.result[phi][:inst]::PhiNode, phi_uses)
+        inst = compact.result[phi]
+        for ur in userefs(inst[:inst]::PhiNode)
+            use = ur[]
+            if isa(use, SSAValue)
```

Comment:
Why not `AnySSAValue`? Could you add a comment here, if applicable?

Reply:
This is post compaction, so there should be no old ssa values left.